### DISCUSSION TITLE
opentelemetry-0.16

### DISCUSSION THREAD

**Discussion by cyrille-leclerc** [jenkinsci/opentelemetry-plugin#142] [General]
* Monitor failures rate when provisioning cloud agents
* Add details on shallow cloning on `git` steps
* Fix [Plugin label metadata in the attributes for some spans is not correct #133](https://github.com/jenkinsci/opentelemetry-plugin/issues/133)

<hr /><em>This discussion was created from the release <a href='https://github.com/jenkinsci/opentelemetry-plugin/releases/tag/opentelemetry-0.16'>opentelemetry-0.16</a>.</em>

**Comment by seumassoltysik** [comment]
Is there a way of starting a new discussion?
I have a question about the demo as I am having trouble getting the fleet-server to run in a healthy state.
For some reason I can't seem to initiate a new discussion in the Discussions section.
